identity/src/main/java/com/android/identity/document/DocumentUtil.kt

```diff
@@ -56,20 +56,19 @@ object DocumentUtil {
     @JvmStatic
     fun managedCredentialHelper(
         document: Document,
-        secureArea: SecureArea?,
-        createKeySettings: CreateKeySettings?,
         domain: String,
+        createCredential: ((Credential?) -> Credential)?,
```

Comment:
I think you can also name the parameter so this could be `((credentialToReplace: Credential?) -> Credential)?`